products/rhcos4/profiles/bsi-2022.profile

```diff
@@ -19,14 +19,16 @@ description: |-
     This baseline implements OS-Level configuration requirements from the following
     sources:
 
+    - Building-Block SYS.1.1 General Server
+    - Building-Block SYS.1.3 Linux Server
     - Building-Block SYS.1.6 Containerisation
     - Building-Block APP.4.4 Kubernetes
 
-    THIS DOES NOT INCLUDE REQUIREMENTS FOR A HARDENED LINUX FROM SYS.1.3 LINUX
-
 selections:
     - bsi_app_4_4:all
     - bsi_sys_1_6:all
```

Comment:
Does it makes sense to keep these two controls here?

I see that the folowwing rules are being added by the kubernetes and containerization control files, maybe they should be migrated to the General and Linux Server control files?
- coreos_enable_selinux_kernel_argument
- selinux_policytype
- selinux_state
- service_firewalld_enabled
- var_selinux_policy_name=targeted
- var_selinux_state=enforcing

Reply:
The same rules are also applied by the general/linux control files. so they are selected multiple times. So technically we could remove app_4_4 and sys_1_6 here without making a difference in the rules. I personally would prefer to leave them here, as we would have more transparency how the controls are adressed.